db/queries/statuses/queries.py

```diff
@@ -12,6 +12,7 @@ def update_application_status(application_id: str):
     """
     application = get_application(application_id)
 
+    # todo(tferns) keep applications in progress if feedback hasnt been completed.
```

Comment:
Working on this, we need to submit the status of feedback with these requests, so we can conditionally set the application to `COMPLETED` if feedback has also been completed.